tools/db_bench_tool.cc

```diff
@@ -604,7 +603,7 @@ DEFINE_bool(use_stderr_info_logger, false,
 
 DEFINE_bool(use_blob_db, false, "Whether to use BlobDB. ");
 
-enum rocksdb::CompressionType StringToCompressionType(const char* ctype) {
+static enum rocksdb::CompressionType StringToCompressionType(const char* ctype) {
```

Comment:
This seems to not encouraged by the Google C++ Style we are following. There seems to be a possible function name conflict to me. I would still put it in a namespace or a unnamed namespace.